api/src/main/java/org/eclipse/microprofile/reactive/messaging/spi/package-info.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2018-2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The MicroProfile Reactive Messaging API Connector SPI
+ * <p>
+ * This package provides the SPI to implement {@code connector}. A {@code connector} can be seen as a Reactive Messaging
+ * plug-in to support a specific messaging technology. For example, you can have a Kafka connector to deal with Kafka,
+ * an AMQP connector to deal interact with AMQP brokers and routers and so on. Connector implementation should be
```

Comment:
deal interact with AMQP brokers and routers and so on => interact with AMQP brokers, routers, and so on.